### PR TITLE
Bump vte to use sixel

### PR DESCRIPTION
**Pull request by junjihashimoto**
This pr modifies vte of nixpkgs to enable sixel.
This is the same as the trial of @gelisam with sixel.

### CODE REVIEW

**Comment by junjihashimoto** [comment]
image

File: .nix-helpers/overlays.nix [hunk 1]
```diff
@@ -34,6 +34,14 @@ let
                     ! any (flip hasPrefix (baseNameOf path)) [ "dist" ".ghc" ];
                 };
 
+              vteWithSixel = self.vte.overrideAttrs ( vself: vsuper: {
+                mesonFlags = vsuper.mesonFlags ++ [ "-Dsixel=true" ];
+                src = super.pkgs.fetchurl {
+                  url = "https://github.com/GNOME/vte/archive/8ef3f6b2f8043d28cbc82520eb094f09333b26ae.tar.gz";
```
Comment (cdepillabout) [inline .nix-helpers/overlays.nix:40]: @junjihashimoto What version of VTE is this?  Is this just `master` as of today?

**Comment by cdepillabout** [comment]
Related to https://github.com/cdepillabout/termonad/pull/219 and https://github.com/cdepillabout/termonad/issues/151

**Comment by cdepillabout** [comment]
@junjihashimoto I pushed two commits.  One commit adds an `enableSixelSupport` Nix option (that is `false` by default), and another commit switches to override the top-level `vte` attribute (so that both the `termonad` and `gi-vte` library get the correct version of vte).

Could you take a look at this and tell me if it doesn't look good?

**Comment by junjihashimoto** [comment]
LGTM! Thx!

**Pull request merged**